lib/ami_spec.rb

```diff
@@ -131,6 +131,12 @@ def self.invoke
 
     options[:tags] = parse_tags(options[:tags])
 
+    options[:amis].each_pair do |role, _|
+      unless Dir.exist?("#{options[:specs]}/#{role}")
+        fail "Role directory #{options[:specs]}/#{role} does not exist. If you'd like to skip the role '#{role}', create the directory but leave it empty (except for a .gitignore file)."
+      end
+    end
```

Comment:
This only tests if we are using the file. We should also test when `options[:ami]` and `options[:role]` are supplied

Reply:
Further up in the file `options[:role] && options[:ami]` are co-erced into `options[:amis]`